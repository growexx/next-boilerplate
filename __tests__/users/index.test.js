import { render } from '@testing-library/react'

import UsersPage from '@src/pages/users'

describe('<UsersPage />', () => {
  it('should match the snapshot', () => {
    const {
      container: { firstChild }
    } = render(<UsersPage />)

    expect(firstChild).toMatchSnapshot()
  })
})
