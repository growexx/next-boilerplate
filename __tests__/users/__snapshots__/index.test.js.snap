// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UsersPage /> should match the snapshot 1`] = `<div />`;
